crates/database/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub mod in_memory_db;
 pub mod states;
 
 #[cfg(feature = "alloydb")]
-pub use alloydb::{AlloyDB, BlockId};
+pub use alloydb::*;
```

Comment:
Supportive, but lets export them one by one. It is nicer to see what is made public.